lib/fast_cache/cache.rb

```diff
@@ -39,6 +39,7 @@ def initialize(max_size, ttl, expire_interval = 100)
       @op_count = 0
       @data = {}
       @expires_at = {}
+      @cleanup = Proc.new if block_given?
```

Comment:
Please provide the block as an `&param`. This type of defaulting is one of the obscure features of Ruby that saves a few characters at the expense of readability.